experimental/benchmarks/node-dc-eis/run-dc-eis.sh

```diff
@@ -16,9 +16,9 @@ MONGODB_PORT=27017
 export DB_URL="mongodb://127.0.0.1:${MONGODB_PORT}/node-dc-eis-db"
 
 #these may need changing when we find out more about the machine we're running on
-NODE_AFFINITY="numactl --physcpubind=0,3"
-MONGO_AFFINITY="numactl --physcpubind=1,2"
-RUNSPEC_AFFINITY="numactl --physcpubind=1,2"
+NODE_AFFINITY="numactl --physcpubind=0,4"
+MONGO_AFFINITY="numactl --physcpubind=1,5"
+RUNSPEC_AFFINITY="numactl --physcpubind=2,6"
```

Comment:
Same here:

I agree I don't think this is correct since I believe there are only 4 CPUs.

Reply:
I guess its ok since it matches what is there for ACME air.  Will go ahead an land since I believe comments for @kunalspathak have been addressed.